packages/firestore/src/local/local_store.ts

```diff
@@ -580,22 +558,21 @@ export class LocalStore {
         // can synthesize remote events when we get permission denied errors while
         // trying to resolve the state of a locally cached document that is in
         // limbo.
-        const remoteVersion = remoteEvent.snapshotVersion;
-        if (!remoteVersion.isEqual(SnapshotVersion.MIN)) {
+        if (!snapshotVersion.isEqual(SnapshotVersion.MIN)) {
```

Comment:
Question: why doesn't this change apply to the Android port?

Reply:
This is just a rename to which allows me to use an object shorthand notation (instead of {snapshotVersion:remoteVersion} I can now use {snapshotVersion}.